frontend/src/__tests__/cypress/cypress/pages/projects.ts

```diff
@@ -74,6 +74,32 @@ class ProjectDetails {
     cy.testA11y();
   }
 
+  findDeletedNotebookImage() {
+    return cy.findByTestId('notebook-image');
+  }
+
+  findNotebookImageName() {
+    return cy.findByTestId('image-display-name');
+  }
+
+  findComponent(componentName: string) {
+    return cy.findByTestId(componentName);
+  }
+
+  findEmptyState(componentName: string) {
+    return this.findComponent(componentName).findByTestId('empty-state-title');
+  }
+
+  shouldDivide() {
+    cy.findAllByTestId('details-page-section').then((sections) => {
+      cy.get('.odh-details-section--divide').then((divider) => {
+        const sectionLength = Cypress.$(sections).length;
+        expect(divider).to.have.length(sectionLength - 1);
+      });
+    });
```

Comment:
When you use a `cy.get` it searches from the page root. In this case you want to search within the scope of each section by using `find`. To do this you can use `cy.wrap` on the jquery result of the first selector to get back into the cypress chain of commands. Then proceed with finding the dividers and chaining an assertion using `should`:
```suggestion
    cy.findAllByTestId('details-page-section').then((sections) => {
      cy.wrap(sections)
        .find('.odh-details-section--divide')
        .should('have.length', sections.length - 1);
    });
```